scripts/website_push_docs.sh

```diff
@@ -34,7 +34,7 @@ git add .
 git commit -q -m "Deploy by $USER"
 
 git remote add heroku git@heroku.com:vagrantup-docs-2.git
-git push -f heroku master
+git push -f heroku main
```

Comment:
I think this should work (https://devcenter.heroku.com/changelog-items/1829) but should give it a go anyway

Reply:
This script is deprecated anyway and we can just remove :smiley:

Reply:
oooo! how about `scripts/website_push_www.sh`?